tests/Doctrine/Tests/Inflector/Rules/Dutch/DutchFunctionalTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Tests\Inflector\Rules\LanguageFunctionalTest;
+
+class DutchFunctionalTest extends LanguageFunctionalTest
+{
+    /**
+     * @return list<array{string, string}>
+     */
+    public function dataSampleWords(): array
+    {
+        return [
+            ['schip', 'schepen'],
+            ['stad', 'steden'],
+            ['gelid', 'gelederen'],
+            // @todo more
+            ['weerman', 'weermannen'],
+            ['ei', 'eieren'],
+            ['rij', 'rijen'],
+            ['mogelijkheid', 'mogelijkheden'],
+            ['adres', 'adressen'],
+            ['olie', 'oliën'],
+            ['industrie', 'industrieën'],
+            ['lid', 'leden'],
+            ['smid', 'smeden'],
+            ['kalf', 'kalveren'],
+            ['lam', 'lammeren'],
+            ['koe', 'koeien'],
+            ['vlo', 'vlooien'],
+            ['leerrede', 'leerredenen'],
+            ['lende', 'lendenen'],
+            ['genius', 'geniën'],
+            ['aanbod', 'aanbiedingen'],
+            ['dank', 'dankbetuigingen'],
+            ['gedrag', 'gedragingen'],
+            ['genot', 'genietingen'],
+            ['lof', 'lofbetuigingen'],
+            ['qaestrices', 'quaestrix'],
+            ['matrices', 'matrix'],
+            ['twitter', 'twitter'],
+            // @todo: multiplitudes array w/o sort order for multiple plural same possibilities
+            ['epos', 'epen'],           
+            // ['epos', 'epossen'],
+            // @todo: multiplitudes array w/o sort order for multiple plural different meenings
+            ['beleg', 'belegeringen'],     // @todo: meening: invest a city and then conquer it
+            // ['beleg', 'beleggen'],      // @todo: meening: call a meeting
+            // ['beleg', 'belegjes'],      // @todo: meening: the slices food on a sandwich
```

Comment:
If the commented code isn't needed anymore, then please remove it.

Reply:
hi you,

i like to keep it in as is now, including meaning correction from meening,
given a singular can have different meaning given the acomplicing plural.
(and yes this hurts the whole inflector idea, context)

ok the notes stay in?

Reply:
hi, i did put this to resolved basic because i had same dialog w/ @greg0ire 
me telling i want the comments in because them are pointers to work to do later-on.

Reply:
What are the next steps that involves those comments that can't be done now? Do you have plans that are currently out of scope for this PR and needs to be postponed for another PR?

Reply:
@SenseException Dutch language is near as complex as German, so it is hard to put all rules down in one.
(i did read somewhere someone in the known to doctrine gave up on German his own language)
My guess that could be weeks maybe months of work to be somewhat complete in rules and exceptions.
Maybe you come to the same conclusion if you follow the out-commented links in the sources in regard,
where several rules are described and explained.

So to be complete in rules and exceptions i think that will not even be possible in just 2 PRs.

So my intent was and is first get this in, and this might just do half or a bit more of the inflections correct.
But also this being a basis for me (and others maybe) to have as basis and to gradually extend the whole.

Maybe this sounds a bit lame and half work done? but given the language is rather complex i'd say
this is the correct aprouch. Any input yours about this thinking very welcome.

This in regard the rules and exceptions.

Your comment is  about the above situation, that does not have any to do w/ the Dutch rules and exceptions,
but, a singular word having multiple total differents in the plural and total differents in the plural meaning.
One is "beleggen" is "call a meeting" and the other is "the food things on you sandwich".
Exact what meaning is intended is clear not by the word itself but it's context as in what words are
around the word, what is the sentence and that does make what meaning so what pluralisation is used.
As far as i know the inflector does not have this capacity itself (me jus doing the dutch not the inflector itself).
The inflector does not look at any context just a word.

On the same subject,
There are also in Dutch singular that has different plurals but with the same meaning.
So any singular such should not return just one plural but an array of possible plurals.
This also is a capacity the inflector does not have (as far as i know)
This should be more easy to implement, return not string but array of strings,
but would be kinda not so usefull in that what plural of the string array would then be picked to use?

Again your thoughts and input very welcome.

Reply:
Yes, "half of the work" in the same scope usually wouldn't be enough for a merge, but it seems that the latest comments are about finishing the other parts.